views/components/HeroCard/index.jsx

```diff
@@ -28,6 +28,8 @@ const HeroCard = () => {
           size="small"
           endIcon={<ArrowForwardOutlined />}
           variant="outlined"
+          href="https://sites.google.com/g.harvard.edu/dpdash/"
+          target="_blank"
```

Comment:
I think we want a `rel` prop here too (`rel="noopener noreferrer"`)

Reply:
[48ad8fe](https://github.com/AMP-SCZ/dpdash/pull/752/commits/48ad8fe702381b975872d919de60de754b994729)